elements/lisk-dpos/src/dpos.ts

```diff
@@ -196,7 +163,10 @@ export class Dpos {
 			delegateActiveRoundLimit,
 		);
 
-		return this.rounds.calcRoundStartHeight(activeRounds);
+		const minBootstrapHeight = this.rounds.lastHeightBootstrap() + 1;
+		const minActiveHeight = this.rounds.calcRoundStartHeight(activeRounds);
```

Comment:
where does activeRounds come from and how it is computed?

Reply:
active rounds comes from the above function `_findEarliestActiveListRound` which was not changed from the previous implementation.